### PR TITLE
Deploy from Operator Hub + NodeObservability + retrieve files

### PR DESCRIPTION
**Pull request by sherine-k**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -11,11 +11,90 @@ git clone https://github.com/openshift/node-observability-operator.git
 IMG=<registry>/<username>/node-observability-operator:latest make container-build container-push deploy
 ```
 
-Deployment from a public release
+### Deployment from a public release
 
-@sherine pls add instructions on operatorhub.io / container catalogue
+1. Create the Operator namespace:
+```sh
+oc create namespace node-observability-operator
```
Comment (josefkarasek) [inline docs/README.md]: oc new-project

Comment (sherine-k) [reply docs/README.md]: done

File: docs/README.md [hunk 2]
```diff
@@ -11,11 +11,90 @@ git clone https://github.com/openshift/node-observability-operator.git
...
+The following `NodeObservability` will create a daemonset targetting all nodes with label `app: example`, running a `node-observability-agent` pod on each of those node using the image given in `.spec.image`.
+
+```yaml
+apiVersion: nodeobservability.olm.openshift.io/v1alpha1
+kind: NodeObservability
+metadata:
+  name: nodeobservability-sample
+  namespace: node-observability-operator
+spec:
+  labels:
```
Comment (josefkarasek) [inline docs/README.md:88]: "node-role.kubernetes.io/worker": ""

Comment (sherine-k) [reply docs/README.md:88]: we agree that the subset of nodes would be the same with or without label `"node-role.kubernetes.io/worker": ""`?
I wanted to show how to select a subset from the worker nodes. maybe `app: example` is not a good one. but couldnt think of anything else we'd be able to use.
I'm happy to put it to `"node-role.kubernetes.io/worker": ""` too...

**Pull request merged**